src/particles/ParticleContainer.js

```diff
@@ -28,16 +28,19 @@ import { hex2rgb } from '../core/utils';
 export default class ParticleContainer extends core.Container
 {
     /**
-     * @param {number} [maxSize=15000] - The maximum number of particles that can be renderer by the container.
+     * @param {number} [maxSize=1500] - The maximum number of particles that can be renderer by the container.
```

Comment:
while you're at it, it looks like this comment should read

> The maximum number of particles that can be rendered by the container.

renderer -> rendered